src/tests/ll_verification.py

```diff
@@ -3544,10 +3546,12 @@ def ll_con_sla_bv_78_c(transport, upperTester, lowerTester, trace):
                                     [ maxPacketTime, maxPacketTime, maxPacketTime, 328, 328, 328, 2120, 2120, 2120, 2120, 2120, 2120 ]):
 
             success = setDataLength(transport, upperTester, initiator.handles[1], txOctets, txTime, trace) and success;
+            requested = success;
+            success = success or (not success and ((txOctets > maxPacketLength) or (txTime > maxPacketTime)));
 
             changed = not ((cmaxTxOctets == min(txOctets, maxPacketLength)) and ((cmaxTxTime == max(txTime, 328)) or (cmaxTxTime == min(txTime, 328))));
```

Comment:
I have a hard time understanding this 'changed' abstraction. How come we need the trailing 'or (cmaxTxTime == min(txTime, 328))' part?

Reply:
That is original code, not changed in this PR.

Reply:
Yes, I realize this. Sorry for being unclear. 
I'm just curious about this, and I mentioned it because I wonder if this is wrong and perhaps then hides other issues re. DLE testing. Let's continue this elsewhere;-)